### ISSUE TITLE
Closing argument list parenthesis on same line isn't detected always

### ISSUE THREAD

**Issue by Rantanen** [Rantanen/Jubjubnest.Style.DotNet#6]
The following code passes despite the closing parenthesis after `error`. Maybe caused by the base constructor call?

```c#
public UpdatedObjectVersionDataResponse(
	ObjectVersionIdentity requestedIdentity,
	ErrorInfo error )
	: base( requestedIdentity, error )
{
```

**Comment by Rantanen** [comment]
Similar issues concerning empty XML tag checks in these places:

```c#
		/// <summary>
		/// Constructs a success response.
		/// </summary>
		/// <param name="requestedIdentity"></param>
		/// <param name="responseData">Response data.</param>
		public FolderDataResponse(
			FolderIdentity requestedIdentity,
			FolderData responseData
		)
			: base( requestedIdentity, responseData )
		{
		}
```

**Comment by Rantanen** [comment]
.... and no XML comments isn't caught either.

This must be a thing that applies to all constructors?

**Comment by Rantanen** [comment]
Fixed in c74d3a4e67aa816a5229d70da529f16264849e4a
